examples/react/modules/tic-tac-toe/components/authenticated.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2018 The boardgame.io Authors
+ *
+ * Use of this source code is governed by a MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+
+import React from 'react';
+import { Client } from 'boardgame.io/react';
+import TicTacToe from '../game';
+import Board from './board';
+import PropTypes from 'prop-types';
+
+const App = Client({
+  game: TicTacToe,
+  board: Board,
+  debug: false,
+  multiplayer: true,
+});
+
+class AuthenticatedClient extends React.Component {
```

Comment:
Looks like the game server this is talking to still uses an unauthenticated game, right? I think it would be more useful to manually create an authenticated game using `db.set` in `examples/react/server.js` to demonstrate that this works (i.e. changing the credentials should result in the move not working; at the moment it seems to be a no-op).

Reply:
I'll look into that. It worked for my manual local testing but there was a lot of tribal knowledge in how to use the two together that I could bake into the tests better.

Reply:
I'm still not sure what the expected behavior you had in mind here is. Right now the game plays just fine regardless of what's in the credentials text field. I would expect that no moves are allowed until you enter the proper credentials in those fields.

Reply:
@nicolodavis This is due to the way that we allow unauthenticated games as the default. 

Once you put the gameID emitted when you start up the examples, then that particular game won't accept moves unless you have the correct credentials.

Until you enter a gameID, you're still in the "default" mode which is fully unauthenticated.

We might be able to move the "create authenticated game" code into the authenticated example itself, which might allow us to prepopulate it.

Reply:
@nicolodavis pushed a change to move authenticated game creation into the authenticated example, which I think will address your concern. Let me know. :)